BlackJackets-ui/src/App.jsx

```diff
@@ -69,7 +79,7 @@ function App() {
             ) : (
               <Route path="*" element={<Navigate to="/login" replace />} />
             )}
-          </Route>
+         
```

Comment:
This `</Route>` should be added back in

Reply:
If i add Route then it is creating issues.